server/app/views/admin/questions/QuestionConfig.java

```diff
@@ -5,13 +5,7 @@
 import com.google.common.base.Preconditions;
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableSet;
-import forms.AddressQuestionForm;
-import forms.EnumeratorQuestionForm;
-import forms.IdQuestionForm;
-import forms.MultiOptionQuestionForm;
-import forms.NumberQuestionForm;
-import forms.QuestionForm;
-import forms.TextQuestionForm;
+import forms.*;
```

Comment:
**nitpick:** revert this change

Reply:
This is the second time its happening. I am wondering if the bin/fmt is doing it or intelliJ